### DISCUSSION TITLE
Custom email routing rules being deleted

### DISCUSSION THREAD

**Discussion by jakeheis** [community/community#68232] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I have a rule that directs all emails coming from my work organization to my work email, while my personal email is the default recipient for all other Github emails. Last week, this custom email routing rule was automatically deleted or lost and I began receiving emails from my work organization in my personal email. Looking at https://github.com/settings/notifications/custom_routing, the rule I created was gone and it just said "No custom routes yet." I recreated the rule and it worked again for a couple of days, and then now again this morning I have the same problem -- I am receiving emails related to the work organization to my personal email account, and the rule is gone. It seems like there is a bug where custom email routing rules are being auto deleted.

**Comment by osteel** [comment]
Same here, my custom route has been deleted twice in the past couple of days

**Reply by jawinn** [reply]
Same for me. My custom routing for an organization disappeared last Friday, Sep 22nd. Yesterday I went into my settings and saw that one of my two routing rules was gone, and re-added it. Today I started getting emails to my personal email again and went to settings and the rule was gone again...

**Reply by ygalblum** [reply]
@jawinn set it again and check. According to the reply I got, it shouldn't happen again (and it hasn't for me since)

**Comment by milindaShehan** [comment]
The best way to proceed is to report the issue to GitHub support. You can do this by submitting a ticket on the GitHub website. Be sure to include as much information as possible in your ticket, such as the following:

- The steps you took to create the custom email routing rule
- The steps you took to recreate the custom email routing rule after it was deleted
- The error message you receive when you try to view or edit your custom email routing rules
- Any other relevant information, such as the email domain of your work organization
- Once you have submitted a ticket, GitHub support will investigate the issue and try to resolve it as soon as possible.

In the meantime, there are a few things you can do to work around the issue:

- You can disable email notifications for all GitHub repositories. To do this, go to your Settings page and click on the Notifications tab. Under Repository notifications, select the Off option.
- You can create a filter in your email client to automatically move emails from GitHub to a specific folder. This will help you to keep track of your GitHub emails and avoid missing any important messages.
- You can use a third-party service to manage your GitHub notifications. There are a number of services available that can do this, such as Pushbullet and Prowler.

**Comment by ygalblum** [comment]
I've opened a ticket to GH support. But the link it private.

**Reply by osteel** [reply]
Thank you

**Reply by quetsalkoatl** [reply]
Thanks. I have the same issue.

**Comment by ygalblum** [comment, marked as answer]
Updated from GH Support:



Hi Ygal,

I've received an update from our engineering team about the issue you're facing.

Our engineering team was working to fix an issue where users' notifications were being incorrectly routed to emails addresses associated with (unaffiliated) organizations they'd left. It appears the script used for this task, which was run multiple times, inadvertently affected custom routes for affiliated organizations as well - specifically where a user isn’t a direct member or owner but is a collaborator. This is likely why your custom route was removed multiple times after you had added it back.

We are now aware of this, and I want to thank you for bringing it to our attention. Changes will be made to the script to prevent this from happening in the future. You should be able to add the route back now without further issues.

We sincerely apologize for any inconvenience this may have caused, and we are grateful for your understanding and patience. If you have any more concerns or questions, please don’t hesitate to reach out.

Regards,
Rocky
